lightning/src/routing/gossip.rs

```diff
@@ -143,7 +143,7 @@ pub struct NetworkGraph<L: Deref> where L::Target: Logger {
 	/// failure so that we don't resync them from gossip. Each SCID is mapped to the instant when we
 	/// removed it from the network graph so that we can forget we removed it after some time (by
 	/// removing it from this map) and it can be resynced from gossip if it appears again.
-	removed_channels: Mutex<HashMap<u64, Instant>>,
+	removed_channels: Mutex<HashMap<u64, u64>>,
```

Comment:
We'll need to remove the `#[cfg(feature = "std")]` above. It's a bit sad that the time types differ now for removed nodes and channels. Tracking removed nodes was a bit more tricky for no-std. I'm wondering now if I should just go ahead and make the change to do that.

Reply:
Right, the std-feature should be removed.

I'm not super-worried about two different timers. My intuition is that this might not be the final combination anyway :)